client/web/src/enterprise/batches/DropdownButton.tsx

```diff
@@ -187,17 +189,18 @@ const DropdownItem: React.FunctionComponent<React.PropsWithChildren<DropdownItem
                 )}
             </H4>
             <Text className="text-wrap text-muted mb-0">
-                {action.type === 'publish' && batchChangesRolloutWindows && batchChangesRolloutWindows.length > 0 ? (
-                    <small>
-                        {action.dropdownDescription} <br />
-                        <b>
-                            Note: Rollout windows have been set up by the admin. This means that some of the selected
-                            changesets won't be processed until a time in the future.
-                        </b>{' '}
-                    </small>
-                ) : (
-                    <small>{action.dropdownDescription}</small>
-                )}
+                <small>
+                    {action.dropdownDescription}
+                    {!loading && shouldDisplayRolloutInfo && (
+                        <>
+                            <br />
+                            <b>
+                                Note: Rollout windows have been set up by the admin. This means that some of the
+                                selected changesets won't be processed until a time in the future.
+                            </b>
```

Comment:
This was feedback from the original PR, but I think we want `<strong>` here instead of `<b>`. `<b>` is mostly for stylistic bold text. It conveys no extra importance or emphasis. `<strong>` should be used for emphasis, as screen readers will convey this as more important text:

```suggestion
                            <strong>
                                Note: Rollout windows have been set up by the admin. This means that some of the
                                selected changesets won't be processed until a time in the future.
                            </strong>
```